### PR TITLE
File display

### PR DESCRIPTION
**Pull request by vfried**
This is to address the issue #400, at the moment it looks like this... 

image

Some things will need to be addressed after this PR is merged:
- See if we can fetch LFS Files or if we can know if a file is from GIT LFS
- Being able to change between branches (now it shows master only)
- Add new formats of files to preview
- Better integration with launching JupyterLab

### CODE REVIEW

File: src/file/Lineage.present.js [hunk 1]
```diff
@@ -49,33 +57,33 @@ class FileLineageGraph extends Component {
   subGraph() {
     const subGraph = new graphlib.Graph()
       .setGraph({})
-      .setDefaultEdgeLabel(function() { return {}; });
+      .setDefaultEdgeLabel(function () { return {}; });
```
Comment (ciyer) [inline src/file/Lineage.present.js]: I think there should be no space between `function` and `()`.

File: src/utils/treeview/data.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
+function buildTree(parts, treeNode, jsonObj) {
+  if (parts.length === 0) {
+    return;
+  }
+  for (var i = 0; i < treeNode.length; i++) {
```
Comment (ciyer) [inline src/utils/treeview/data.js]: `var` -> `let`

File: src/utils/treeview/data.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    if (parts[0] == treeNode[i].name) {
+      buildTree(parts.splice(1, parts.length), treeNode[i].children, jsonObj);
+      return;
+    }
+  }
+  var newNode;
```
Comment (ciyer) [inline src/utils/treeview/data.js]: `var` -> `let`

File: src/utils/treeview/data.js (hunk 2 above)
Comment (vfried) [reply src/utils/treeview/data.js]: I deleted this file, the data was just for testing

File: src/utils/treeview/data.js (hunk 3 above)
Comment (vfried) [reply src/utils/treeview/data.js]: This file was deleted, i just used it for testing reasons.

File: src/file/File.container.js [hunk 4]
```diff
@@ -85,10 +110,12 @@ class FilePreview extends React.Component {
   render(){
     // File has not yet been fetched
     if (!this.props.file) {
-      return <p>Loading...</p>;
+      return "Loading...";
     }
     // Various types of images
     if (this.fileIsImage()) {
+      if(atobUTF8(this.props.file.content).includes("https://git-lfs.github.com/"))
```
Comment (ciyer) [inline src/file/File.container.js]: I think we should just use regular atob here, since the image is going to be binary content, not UTF-8 encoded.

Comment (vfried) [reply src/file/File.container.js]: I think you are right, i am not parsing the image in that line, I just check if the content contains the git lfs text but that will not have special characters anyway

**Pull request merged**